docs/template/public/less/_variables.less

```diff
@@ -22,4 +22,11 @@
 @dark-blue: #27394f;
 @side-nav-link: #dbe3eb;
 @top-nav-hover: #0376C6;
-@btn-type-color: #0b0080;
\ No newline at end of file
+@btn-type-color: #0b0080;
+@kss-green: #04ce26;
```

Comment:
We really should separate kss (techne page) variables form the actuall framework